tests/functional/spec/components/checkbox-mutually-exclusive.spec.js

```diff
@@ -1,62 +1,94 @@
-const helpers = require('../../helpers');
+const helpers = require("../../helpers");
 
-const MandatoryCheckboxPage = require('../../pages/components/checkbox/mutually-exclusive/mandatory-checkbox.page');
-const SummaryPage = require('../../pages/components/checkbox/mutually-exclusive/summary.page');
+const MandatoryCheckboxPage = require("../../pages/components/checkbox/mutually-exclusive/mandatory-checkbox.page");
+const SummaryPage = require("../../pages/components/checkbox/mutually-exclusive/summary.page");
 
-describe('Component: Mutually Exclusive Checkbox', function() {
+describe("Component: Mutually Exclusive Checkbox", function() {
 
   beforeEach(function() {
-    return helpers.openQuestionnaire('test_checkbox_mutually_exclusive.json').then(() => {
-      return browser
-        .getUrl().should.eventually.contain(MandatoryCheckboxPage.pageName);
-    });
+    return helpers.openQuestionnaire("test_checkbox_mutually_exclusive.json");
   });
 
-  describe('Given a mutually exclusive option is available', function() {
-    it('When the user clicks a range of options and the mutually exclusive option then an error should be visible.', function() {
+  describe("Given the user has clicked multiple non-exclusive options", function() {
+    it("When then user clicks the mutually exclusive option, Then only the mutually exclusive option should be on the summary page.", function() {
+
       return browser
+        // Given
         .click(MandatoryCheckboxPage.cheese())
         .click(MandatoryCheckboxPage.ham())
         .click(MandatoryCheckboxPage.pineapple())
+
+        // When
         .click(MandatoryCheckboxPage.none())
         .click(MandatoryCheckboxPage.submit())
```

Comment:
We should check after the exclusive value is selected that the others are deselected rather than just relying on the Summary page not including them.

Reply:
@dwyeradam Ready for re-review.